### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by psychogenic** [bicarlsen/easy-scpi#11]
Greetings Brian,

Couldn't find how to get in touch (linked-in won't let me, twitter account from there is down) so am using this.  Not writing about an issue, just some thanks.

I've been using easy-scpi to build wrappers from the devices I have and use the most and it made everything extremely easy.  

I made some classes to give the objects derived from scpi.Instrument utility functions (and autocomplete!) so I don't have to remember each thing's API.  I've been using this stuff for a while, but as I was doing a demo of it on a recent livestream I decided it might be useful to others.

If you're curious how I'm using the lib, its the Rigol DSO, Unitrend signal generator and Siglent power supply support in: https://github.com/psychogenic/psytestbench 

In any case, thanks again.
Cheers,
Pat D

**Comment by bicarlsen** [comment]
Man, thank you so much for the message. It's so cool to hear that it's been useful for you, and the project looks really nice as well.

**Comment by psychogenic** [comment]
:)  Awesome: It's good knowing that some people use your stuff, but actually getting some feedback is nicer still, so yeah, thanks again!  

I finally made a public announcement and did a bit of an overview of easy-scpi and the drivers I built on top.  If you want to see it:
https://www.youtube.com/watch?v=xko8wzEBqfc

Am going to let this sit for a bit, see if anyone comes up with bugs or feature requests, but one thing I want to deal with at some point is binary xfers (e.g. display dumps), so I'll be looking at if this is already supported here or probably doing a PR if not (though this will probably be later in July).

Cheerio,
Pat D
